bin/options.js

```diff
@@ -40,8 +40,8 @@ function getOptions() {
     process.argv = process.argv
       .slice(0, 2)
       .concat(opts.concat(process.argv.slice(2)));
-  } catch (err) {
-    // ignore
+  } catch (ignore) {
+    // NOTE: should console.error() and throw the error
```

Comment:
What did you mean by this comment?

Reply:
Exactly what it says. Believe it's a shortcoming of existing code. If anything goes wrong parsing the "mocha.opts" file or splicing its pieces/parts back into the cmdline arguments, the error is silently discarded. This just seems wrong, so I made a note; figured it should at least be discussed (and hopefully fixed in different issue).

Reply:
@plroebuck I got it. Thank you for the explanation.

Reply:
Yeah, I think we should be emitting a warning here, at minimum--unless there's some Git or issue history that makes me think otherwise.

That env var should be set anyway, because it's only used to check whether or not `getOptions()` should be called.  

But that's another issue.

Reply:
Agreed.